multicluster/config/default/configmap/controller_manager_config.yaml

```diff
@@ -13,3 +13,4 @@ podCIDRs:
   - ""
 gatewayIPPrecedence: "private"
 endpointIPType: "ClusterIP"
+enableStretchedNetworkPolicy: true
```

Comment:
I am not sure about this default value, @jianjuns any suggestion?

Reply:
I would default to "false".

Reply:
Done.